### PR TITLE
Refactor Tuya Remote quirks

### PR DESCRIPTION
**Pull request by Adminiuga**
Refactor Tuya smart remote quirks. Maintains compatibility with the original quirk and device triggers.
The original quirk manipulated raw messages by overwriting the command ID based on command argument. While clever, I don't think this was the correct approach in zigpy terms and was causing some extra noise in the logs (data remains). 

This PR defines proper "cluster specific" command for press types and fire events depending on the press type.

Add signature for Benexmart Tuya remotes: [2 gang](https://www.aliexpress.com/item/4001230762751.html) and [3 gang](https://www.aliexpress.com/item/4001230762751.html)

@dmulcahey let me know if you prefer this PR to be split in two

### CODE REVIEW

**Comment by Adminiuga** [comment]
I don't get why it fails here. I don't see this module at all in the repo
```
zhaquirks/__init__.py:369: in <module>
    importlib.import_module(modname)
zhaquirks/tuya/ts0041_zemismart.py:6: in <module>
    from . import TuyaSmartRemote, TuyaSmartRemoteOnOffCluster
E   ImportError: cannot import name 'TuyaSmartRemote' from 'zhaquirks.tuya' (/home/runner/work/zha-device-handlers/zha-device-handlers/zhaquirks/tuya/__init__.py)
Error: Process completed with exit code 4.
```

File: zhaquirks/tuya/__init__.py [hunk 1]
```diff
@@ -354,47 +354,36 @@ def __init__(self, *args, **kwargs):
...
-class TuyaSmartRemoteOnOffCluster(EventableCluster):
-    """TuyaSmartRemoteOnOffCluster: this cluster manipulates messages from the remote control and converts them to command_ids."""
+class TuyaSmartRemoteOnOffCluster(OnOff, EventableCluster):
+    """TuyaSmartRemoteOnOffCluster: fire events corresponding to press type."""
 
-    cluster_id = 0x0006
+    press_type = {
+        0x00: SHORT_PRESS,
+        0x01: DOUBLE_PRESS,
+        0x03: LONG_PRESS,
```
Comment (dmulcahey) [inline zhaquirks/tuya/__init__.py:363]: This was 0x02 previously. Is this change intentional?

Comment (Adminiuga) [reply zhaquirks/tuya/__init__.py:363]: Good catch. I'll fix it.

Comment (Adminiuga) [reply zhaquirks/tuya/__init__.py]: fixed in https://github.com/zigpy/zha-device-handlers/pull/673/commits/14ac22d66f6033e0fc9340984676edc2c645b057

**Comment by Adminiuga** [comment]
Updated Long press to 0x02 code.

**Comment by DavidFFerreira** [comment]
@JbPasquier  I did try to merge your code with my HA, and the _TZ3400_keyjhapk still dont work
